Sources/SWBWindowsPlatform/Specs/WindowsLd.xcspec

```diff
@@ -52,6 +52,18 @@
                 CommandLineFlag = "--sysroot";
                 IsInputDependency = Yes;
             },
+            {
+                Name = _LD_MULTIARCH;
+                Type = Boolean;
+                DefaultValue = NO;
+                Condition = "$(ALTERNATE_LINKER) == link";
+            },
+            {
+                Name = _LD_MULTIARCH_PREFIX_MAP;
+                Type = StringList;
+                DefaultValue = "x86_64:x64 aarch64:arm64 i386:x86 thumbv7:arm";
```

Comment:
Sorry for the churn but I changed thumbv7 to armv7 in #304. And i386 should be i686.